src/config/urls.py

```diff
@@ -15,9 +15,11 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from app import views as app_views
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', include('app.urls'))
+    path('', include('app.urls')),
+    path('errands/', app_views.errands)
```

Comment:
This should be moved to `src/app/urls.py` since it's specific to the app instead of the config.